include/pacmod_game_control/globals.h

```diff
@@ -40,11 +40,11 @@ enum TurnSignalState
   SIGNAL_HAZARD
 };
 
-enum DoorSignalState
+enum RearPassDoorState
 {
-  SLIDING_DOOR_NEUTRAL,
-  SLIDING_DOOR_OPEN,
-  SLIDING_DOOR_CLOSE
+  REAR_PASS_DOOR_NEUTRAL,
+  REAR_PASS_DOOR_OPEN,
+  REAR_PASS_DOOR_CLOSE
 };
```

Comment:
I recommend dropping this and using the values that are in the ROS Message instead. So, below, instead of using REAR_PASS_DOOR_NETURAL, use `pacmod_msgs::SystemCmdInt::DOOR_NEUTRAL` (as added in astuff/astuff_sensor_msgs#38 ).

Reply:
would it be okay then to have different format than other systems, like enum shift or enum turn?

Reply:
or can I change the values in other systems as well?

Reply:
If you would like to change the other systems, I'm OK with that too. It's better to use the values that are in the message instead of re-defining them in enums here. If you don't want to do it right now, I'll get it in my clean-up of this package in the future.

Reply:
yes, I can do it right now!